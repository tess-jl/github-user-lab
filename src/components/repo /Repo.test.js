import React from 'react';
import PropTypes from 'prop-types';

const Search = () => {

  return (

  );
}

Search.propTypes = {

};

export default Search; 

