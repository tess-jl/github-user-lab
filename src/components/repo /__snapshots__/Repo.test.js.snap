// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo component renders Repo 1`] = `
<section>
  <p />
</section>
`;
